.github/workflows/pytest.yml

```diff
@@ -99,10 +108,7 @@ jobs:
           upload_secret_key: ${{ secrets.QUERIES_UPLOAD_SECRET }}
         if: github.event_name == 'push' && github.repository == 'saleor/saleor'
 
-      # Run linters and Django related checks
-      # `git config` command is a workaround for https://github.com/actions/runner-images/issues/6775
-      - name: Run Linters and Checks
+      - name: Run E2E tests
         run: |
-          git config --global --add safe.directory "$GITHUB_WORKSPACE"
-          pre-commit run --all
-        if: ${{ always() }}
+          pytest \
+            -m "e2e"
```

Comment:
Is it intended that it is excluded from coverage?

Reply:
Yes. In my opinion, we shouldn't sum coverage from the E2E test and unit tests. 
If we want to have coverage on the E2E test we should ad it as a separate metric.